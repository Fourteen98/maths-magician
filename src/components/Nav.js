function Navbar() {
  return (
    <nav className="navbar">
      <h1> Math Magicians </h1>
      <div className="links">
        <a href="/">Home</a>
        <a href="/App">Calculator</a>
        <a href="/Quote">Quote</a>
      </div>
    </nav>
  );
}
export default Navbar;
