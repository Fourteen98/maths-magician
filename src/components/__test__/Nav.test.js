import React from 'react';
import renderer from 'react-test-renderer';
import Nav from '../Nav';

it('renders to match the snapshot', () => {
const tree = renderer.create(<Nav />).toJSON();
expect(tree).toMatchSnapshot();
}); 