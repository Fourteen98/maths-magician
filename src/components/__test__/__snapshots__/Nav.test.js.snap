// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders to match the snapshot 1`] = `
<nav
  className="navbar"
>
  <h1>
     Math Magicians 
  </h1>
  <div
    className="links"
  >
    <a
      href="/"
    >
      Home
    </a>
    <a
      href="/App"
    >
      Calculator
    </a>
    <a
      href="/Quote"
    >
      Quote
    </a>
  </div>
</nav>
`;
